tests/conftest.py

```diff
@@ -17,3 +21,26 @@ def user():
     return User.objects.create_user(
         email="alice@test.com", password="password", name="Alice"
     )
+
+
+# set these values to have messages generated by tests appear in a test channel
+slack_token = os.environ.get("SLACK_BOT_TOKEN")
+slack_test_channel = os.environ.get("SLACK_TEST_CHANNEL")
```

Comment:
IIRC this project is already installing environs, would be nice to be consistent across the project and it lets us specify the expected datatype too.

Reply:
Yeah, this is more of a quick trick that developers can do, rather than formal config. It's taken from job-server https://github.com/opensafely-core/job-server/blob/main/tests/conftest.py#L85